internal/ethapi/api.go

```diff
@@ -499,9 +499,6 @@ func (s *PersonalAccountAPI) SignTransaction(ctx context.Context, args Transacti
 	if args.GasPrice == nil && (args.MaxFeePerGas == nil || args.MaxPriorityFeePerGas == nil) {
 		return nil, errors.New("missing gasPrice or maxFeePerGas/maxPriorityFeePerGas")
 	}
-	if args.IsEIP4844() {
-		return nil, errBlobTxNotSupported
```

Comment:
Ah this is annoying. Can we keep this check somehow if the signer is not "external"?

Reply:
Why?

Reply:
In the vein of deprecating Sign/SendTransaction methods I think we shouldn't add new features (i.e. here blob txes).

Reply:
Oh right. So, on personal, I think we can leave this be. That is: my change here can be undone